crates/key-server/src/master_keys.rs

```diff
@@ -122,7 +152,9 @@ impl MasterKeys {
         package_id: &ObjectID,
     ) -> anyhow::Result<&IbeMasterKey, InternalError> {
         match self {
-            MasterKeys::Open { master_key } => Ok(master_key),
+            MasterKeys::Open { master_key } | MasterKeys::Committee { master_key, .. } => {
```

Comment:
what if we already moved to the new committee?

Reply:
i left the next committee code change intentionally in TODO and out of this PR to keep this pr small. do you like to review it altogether instead?

the way it I plan to set up is: if a next master key is provided, load it in memory and start server. pull for the committee state for the next committee id till the state finalized (the current committee obj is now deleted too), then use MasterKeys to use next master key to serve any request.